CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add missing data source to atlas dashboards.
 - Fix missing provider specific dashboards.
+- Upgrade old angular plugins in atlas dashboards.
```

Comment:
This would make more sense IMO:
```suggestion
- Udate alertmanager-overview dashboard (angular deprecation).
```

Reply:
It would but I upgraded a bunch of others that I forgot to mention here. Let me fix it

Reply:
```suggestion
- Update alertmanager-overview dashboard (angular deprecation).
- Update fluentbit dashboard (angular deprecation).
- Update prometheus-overview dashboard (angular deprecation).
- Update prometheus-remote-write dashboard (angular deprecation).
```